### ISSUE TITLE
Scenes command xx% for blinds, switches turn to 100% no matter what

### ISSUE THREAD

**Issue by MarcelBaas** [domoticz/domoticz#5365]
Goodmorning, this morning I received another update "build 14549" However since two updated last Saturday evening al my blinds do not respond correct to en open command from the created scenes. The scene should give 50%  some blinds turn completely others just stay closed. Closing the blinds looks ok.
Already rebooted the Raspberry, without "luck". 
What I noticed is that all blind switches (in Domoticz) are set to 100% instead of the percentage commanded from the scene. To me it looks like something has changed in the connection between the scenes and the blind switches inside domoticz 
Nice to know: the blinds have been working fine for the last 3 years ;)

Everything is controlled by MQTT

Regards
Marcel

**Comment by gizmocuz** [comment]
@MarcelBaas , there have been a big change in blinds... Blinds inverted has been removed and for each blind you can now specify to 'reverse the state (open/close) and/or 'reverse position (0-100'
Also the blinds are now acting as other global hardware/software does, open=100, close=0

What kind of hardware device is your blind?

I tested the blind in a Scene, setting the blind here to 32%.
Pressing the left button (turn on scene) will set it to 32% open, pressing the right button (turn off scene) will close the blind.
I hope this is the desired behavior.

When you operate the blinds in the GUI all is working correct?

**Comment by ajarzyn** [comment]
Hello @gizmocuz,

I've tested @MarcelBaas scenario and in my case the problem exists to.
Following settings opens the gate to 100%:
![image](https://user-images.githubusercontent.com/18590439/194816024-077ebf20-eb13-45db-8f31-adf9b68e5add.png)

And following:
![image](https://user-images.githubusercontent.com/18590439/194816241-08336dc9-96f1-40d8-84c0-997b13cccf12.png)

closes the gate to 0%.

It looks like `%` are  just ignored.

**Comment by MarcelBaas** [comment]
Hey,  Just received another update and.... we're back in business :-). THX!!! Sorry had not seen your question upto  just now. 
If it helps for future releases I'm using homemade units with and ESP8266. MQTT receives the message from Domoticz. And the units are listening to the corresponding ID. 

But as I stated before the problem was not in the switches within Domoticz. It was somewhere in the connection between the scenes and the switches.  I just tested some of the blinds they work fine.  version 2022.1 build 14558 

A bit of a bummer (sorry :S) I just noticed that there is an issue with the Tuya blinds. This is something with the switches in Domoticz. Let me close this issue. Then I'll do some investigation on the Tuya stuff. Once it's more clear when is goed wrong where I'll open a new issue for the Tuya units.

Best Reagrds
Marcel Baas
